frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
+
+class API {
+    BASE_URL = '/api';
+    myFetch = (url, config) =>
+        window.fetch(this.BASE_URL + url, {
+            headers: { 'content-type': 'application/json' },
+            ...config,
+        });
+
+    async login({ pin, phoneNumber }) {
+        try {
+            return await this.myFetch(`/users/login`, {
+                method: 'POST',
+                body: JSON.stringify({
+                    pin,
+                    phoneNumber,
+                }),
+            }).then((res) => res.json());
+        } catch (error) {
+            throw error;
+        }
+    }
+
+    async fetchCourses(config) {
+        try {
+            const queries = Object.keys(config)
```

Comment:
Would be good to use some kind of URL handling library to ensure everything works okay with characters that need to be escaped.

Reply:
Good idea. I will wrap that in encodeUriComponent()